src/commands/ticket_command.py

```diff
@@ -0,0 +1,37 @@
+from ticket_class import Ticket
+
+async def ticket_command(self, customer, channel, guild, ticket_channel):
+    ticket_message = f'Please describe the problem in one message and a moderator will be contacted'
+    
+    def conv_check(m):
+        return m.author == customer and m.channel == channel
+
+    await channel.send(str(ticket_message))
+
+    description = await self.bot.wait_for("message", check=conv_check)
+      
+    customer_ticket = Ticket(111, customer, ticket_channel, description)
```

Comment:
What was the library you mentioned that would be useful to create unique ticket numbers?